build/charts/antrea-windows/values.yaml

```diff
@@ -0,0 +1,7 @@
+# -- Container image used for Antrea Windows Nodes
+image:
+  repository: "antrea/antrea-windows"
+  tag: "latest"
+
+# -- Enable running Windows OVS processes inside antrea-ovs container in antrea-agent Pod
+includeOVS: false
```

Comment:
By default, when the includeOVS flag is set to false in the Helm chart values, it means that Windows Open vSwitch (OVS) processes inside the antrea-ovs container won't start automatically when deploying with Helm. However, OVS processes are essential for Antrea's operation. While it's acceptable to have includeOVS set to false by default for flexibility, it's crucial to explicitly mention in the documentation that when deploying with Helm, users must set includeOVS to true. The current pull request (PR) does not contain documentation updates regarding this setting.

Reply:
I am fine with defaulting `includeOVS` to `true` in this PR, but it doesn't matter much: this PR introduces Helm templating to generate the Antrea Windows manifests, _but at the moment we are not exposing this Helm chart to users_. For the same reason, there is no need to update documentation as part of this PR.

We may start publishing the Helm chart in a subsequent release.

Reply:
Defaulting includeOVS to true makes sense,
If the Helm chart is not currently exposed to users in current release then it would not be necessary to add documentation in this PR. Anyway We will revisit the documentation and user-facing aspects when we start considering publishing the Helm chart in a subsequent release.

Reply:
@shikharish Could you default `includeOVS` to `true` and make the necessary changes for manifest generation? I think that going forward most users will let Antrea run OVS daemons inside a container, and when we do expose the chart to users, a default value of `true` will therefore make more sense.
Please use a separate commit for that change, for easier reviewing.

Reply:
@antoninbas So `generate-manifest-windows.sh` by default should create the manifest with OVS enabled?

Reply:
I think you can keep the same behavior for `generate-manifest-windows.sh` as now. But if the default value is changed for the chart, then the implementation of the script needs to change a little bit. In particular, if `--include-ovs` is not provided, you will need to explicitly set the chart value to `false`. I think the best thing to do in the script would be to always _explicitly_ set the value based on the value of `$INCLUDE_OVS`.